userscript-styles/WKED-self-study.css

```
@@ -12,72 +12,72 @@
 /* See https://community.wanikani.com/t/userscript-self-study-quiz/13191 */
 
 @-moz-document domain("www.wanikani.com") {
-#wkof_ds #ss_quiz {
-  border-color: var(--ED-surface-2, #282828);
-  background-color: var(--ED-surface-2, #282828);
-}
+  #wkof_ds #ss_quiz {
+    border-color: var(--ED-surface-2, #282828);
+    background-color: var(--ED-surface-2, #282828);
+  }
 
-#wkof_ds #ss_quiz .titlebar {
-  background-color: var(--ED-surface-2, #282828);
-  color: var(--ED-text-color, #eeeeee);
-}
+  #wkof_ds #ss_quiz .titlebar {
+    background-color: var(--ED-surface-2, #282828);
+    color: var(--ED-text-color, #eeeeee);
+  }
 
-#wkof_ds #ss_quiz .statusbar {
-  background-color: var(--ED-surface-2, #282828);
-  color: var(--ED-text-color, #eeeeee);
-}
+  #wkof_ds #ss_quiz .statusbar {
+    background-color: var(--ED-surface-2, #282828);
+    color: var(--ED-text-color, #eeeeee);
+  }
 
-#wkof_ds #ss_quiz .answer {
-  background-color: var(--ED-surface-3, #303030);
-}
+  #wkof_ds #ss_quiz .answer {
+    background-color: var(--ED-surface-3, #303030);
+  }
 
-#wkof_ds #ss_quiz .answer input {
-  background-color: var(--ED-surface-5, #bababa);
-  border-color: var(--ED-surface-4, #535353);
-  --ED-text-color: var(--ED-inverted-text, #151515);
-}
+  #wkof_ds #ss_quiz .answer input {
+    background-color: var(--ED-surface-5, #bababa);
```

`--ED-surface-5` no longer exists
```suggestion
    background-color: var(--ED-surface-inv, #bababa);
```